app/main.py

```diff
@@ -1,7 +1,34 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+DEFAULT_CITY = "Paris"
+
+
+def get_weather(city: str) -> None:
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError(
+            "API_KEY is not set. Please provide it as an environment variable"
+        )
+
+    url = f"{BASE_URL}?key={api_key}&q={DEFAULT_CITY}"
```

Comment:
The `get_weather` function should use the `city` parameter instead of the `DEFAULT_CITY` constant. Update the URL construction to use the `city` parameter.